build-conventions/src/main/java/org/elasticsearch/gradle/internal/conventions/util/Util.java

```diff
@@ -119,29 +119,29 @@ public static SourceSetContainer getJavaSourceSets(Project project) {
         return project.getExtensions().getByType(JavaPluginExtension.class).getSourceSets();
     }
 
-        public static File locateElasticsearchWorkspace(Gradle gradle) {
-            if(gradle.getRootProject().getName().startsWith("build-tools")) {
-                File buildToolsParent = gradle.getRootProject().getRootDir().getParentFile();
-                if(versionFileExists(buildToolsParent)) {
-                    return buildToolsParent;
-                }
+    public static File locateElasticsearchWorkspace(Gradle gradle) {
```

Comment:
All I did was fix the indenting here. Not sure why GitHub is rendering this diff like a maniac.